src/__tests__/ReactWrapper-spec.js

```diff
@@ -332,6 +332,43 @@ describeWithDOM('mount', () => {
       expect(spy.args[0][0].someSpecialData).to.equal('foo');
     });
 
+    describe('Normalizing JS event names', () => {
+      it('should convert lowercase events to React camelcase', () => {
+        const spy = sinon.spy();
+        class Foo extends React.Component {
+          render() {
+            return (
+              <a onDoubleClick={spy}>foo</a>
+            );
+          }
+        }
+
+        const wrapper = mount(<Foo />);
+
+        wrapper.simulate('doubleclick');
+        expect(spy.calledOnce).to.be.true;
+      });
+
+      describeIf(!REACT013, 'normalizing mouseenter', () => {
+        it('should convert lowercase events to React camelcase', () => {
+          const spy = sinon.spy();
+          class Foo extends React.Component {
+            render() {
+              return (
+                <a onMouseEnter={spy}>foo</a>
+              );
+            }
+          }
+
+          const wrapper = mount(<Foo />);
+
+          wrapper.simulate('mouseenter');
+          expect(spy.calledOnce).to.be.true;
```

Comment:
You'll need to pull from master and changes these assertions to be `to.equal(true)`. A change was merged that causes a linting error with these types of assertions!